internal/runnerinstall/k8s.go

```diff
@@ -101,22 +100,25 @@ func (i *K8sRunnerInstaller) Install(ctx context.Context, opts *InstallOpts) err
 
 	version := i.Config.Version
 	if version == "" {
-		version = helminstallutil.DefaultHelmChartVersion
+		tags, err := helminstallutil.GetLatestHelmChartVersion(ctx)
+		if err != nil {
+			opts.UI.Output("Error getting latest tag of Waypoint helm chart.", terminal.WithErrorStyle())
+			return err
+		}
+		version = *tags[0].Name
 	}
 
 	s.Update("Locating chart...")
-	path, err := client.LocateChart("https://github.com/hashicorp/waypoint-helm/archive/refs/tags/v"+version+".tar.gz", settings)
+	path, err := client.LocateChart("https://github.com/hashicorp/waypoint-helm/archive/refs/tags/"+version+".tar.gz", settings)
```

Comment:
Does it matter for this API if the `v` is in front of the version or not? I assume so because it's looking for a specific tag. We should add a note of the expected format to the `k8s-helm-version` flag description, both here and on `server install` flag.